Test/dafny0/PrintEffects.dfy

```diff
@@ -1,5 +1,5 @@
-// RUN: %dafny /compile:3 "%s" > "%t"
-// RUN: %dafny_0 /compile:3 /trackPrintEffects:1 "%s" >> "%t"
+// RUN: %baredafny run %args "%s" > "%t"
+// RUN: %baredafny run %args --track-print-effects "%s" >> "%t" || true
```

Comment:
```suggestion
// RUN: ! %baredafny run %args --track-print-effects "%s" >> "%t"
```

Not just tidier, strictly more precise since it will fail if the exit code is 0.

Reply:
Applied this across all tests